org.recoll.recoll.yml

```diff
@@ -0,0 +1,125 @@
+id: org.recoll.recoll
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+base: io.qt.qtwebkit.BaseApp
+base-version: 5.15-23.08
+command: recoll
+rename-desktop-file: recoll-searchgui.desktop
+rename-icon: recoll
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=home
+  - --env=ASPELL_CONF=dict-dir /app/share/dicts; local-data-dir /app/share/datadicts
```

Comment:
this seems like an odd format, does this work?

Reply:
I have tested it, and it works as intended.